packages/material-ui-lab/src/TreeItem/TreeItem.test.js

```diff
@@ -519,7 +519,7 @@ describe('<TreeItem />', () => {
                 <button
                   data-testid="button"
                   type="button"
-                  onClick={() => setState(value => !value)}
+                  onClick={() => setState((value) => !value)}
```

Comment:
This is not related to the issue but it's a result of me running `yarn prettier:all`

Reply:
Committed on master.